### PR TITLE
06-TimeManagement: implement homework with system timer, timer, absol…

### PR DESCRIPTION
**Pull request by d-el**
Time Management:
- system timer
- timer
- absolute time

### CODE REVIEW

File: 06-TimeManagement/itime.c [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+	n = sscanf(pbuf, "%u", &ms);
+	if (n < 1) {
+		pr_info("itime: %s: error time\n", __func__);
+		return count;
+	}
+
+	if (ms < 100)
+		ms = 100;
+	if (ms > 10000)
+		ms = 10000;
```
Comment (DevyatovAndrey) [inline 06-TimeManagement/itime.c]: Is there any way to stop the timer?

Comment (d-el) [reply 06-TimeManagement/itime.c]: ?
Its limiter.

Comment (d-el) [reply 06-TimeManagement/itime.c]: No. add?

Comment (DevyatovAndrey) [reply 06-TimeManagement/itime.c]: Yes, please.
Since without this feature this timer will overflood the log.

Comment (d-el) [reply 06-TimeManagement/itime.c]: Done.

**Pull request merged**